src/component/PowerQuery/index.tsx

```diff
@@ -59,6 +61,8 @@ const PowerQuery = ({ onSubmit, enableSubmit }: PowerQueryProps) => {
     includeRecipes: true,
     includeCreative: true,
     multiplayer: false,
+    noHerba: true,
```

Comment:
Please make this `false` by default. If someone were looking up a recipe for Title or Sparkling powers, and they got no results, they might not realize they need to uncheck this option.

Reply:
done